services/core-web/src/components/Forms/noticeOfWork/UpdateNOWInspectorsForm.js

```diff
@@ -32,17 +35,59 @@ const UpdateNOWLeadInspectorForm = (props) => {
             name="lead_inspector_party_guid"
             label={
               !props.isAdminView
-                ? "Assign a lead inspector before continuing. This assignment can be updated later under the Administrative tab."
+                ? "Assign the Lead Inspector before continuing. This assignment can be updated later under the Administrative tab."
                 : ""
             }
             component={renderConfig.GROUPED_SELECT}
-            placeholder="Start typing lead inspectors name"
+            placeholder="Start typing the Lead Inspector's name"
             validate={[required]}
             data={props.inspectors}
             disabled={!props.isEditMode}
             onSelect={props.setLeadInspectorPartyGuid}
           />
         </Form.Item>
+        <div className="field-title">Issuing Inspector</div>
+        <Form.Item>
+          <Field
+            id="issuing_inspector_party_guid"
+            name="issuing_inspector_party_guid"
+            label={
+              !props.isAdminView
+                ? "Optionally assign the Issuing Inspector before continuing. This assignment can be updated later under the Administrative tab."
+                : ""
+            }
+            component={renderConfig.GROUPED_SELECT}
+            placeholder="Start typing the Issuing Inspector's name"
+            data={props.inspectors}
+            disabled={!props.isEditMode}
+            onSelect={props.setIssuingInspectorPartyGuid}
+          />
+        </Form.Item>
+        {!props.isEditMode && (
+          <>
+            {props.noticeOfWork?.issuing_inspector?.signature ? (
+              <img
+                src={props.noticeOfWork?.issuing_inspector?.signature}
+                alt="Signature"
```

Comment:
Whats going on with the question marks here?  I'm not familiar with this syntax
`props.noticeOfWork?.issuing_inspector?.signature`

Reply:
This is a clever null check.. why haven't we been using this everywhere

Reply:
also, if you already checked its existence on line 69, can you just pass it to the src?

Reply:
> Just googled it - This is a clever null check.. why haven't we been using this everywhere

I know right!! I learned about it the other night... We definitely should start using it. Sounds good!

Reply:
I'm absolutely going to start using this!